tableaudocumentapi/connection.py

```diff
@@ -133,3 +135,17 @@ def dbclass(self, value):
 
         self._class = value
         self._connectionXML.set('class', value)
+
+    ###########
+    # port
+    ###########
+    @property
+    def port(self):
+        return self._port
+
+    @port.setter
+    def port(self, value):
+        self._port = value
+        # If port is None we don't actually want to update the XML
+        if value is not None:
+            self._connectionXML.set('port', value)
```

Comment:
Done -- one SO post said del doesn't work, but others said it's fine and it works for me too